docs/compendium/substitute.md

```diff
@@ -1,7 +1,7 @@
 # Substitution
 
 Substitution is the combination of an base compendium, "base" for short, and an overlay compendium, or "overlay".
-A user can choose files from the overlay to replace files of the base, or upload new files.
+A user can choose files from the overlay to replace files of the base, or upload new files. Additionally the user can choose, if the metadata of the base ERC will be adopted for substitution (`keepBase`) or there will be a new extraction of the metadata for the substituted ERC. This new extraction is divided into two choices. The user can let the new extracted metadata be merged into the existing metadata of the base ERC (`extractAndMerge` - **not implemented**) or just save the extracted metadata (`extract` - **not implemented**).
```

Comment:
Please put each sentence on its own line, makes change tracking and merging easier and has no impact on the rendered output.